src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml

```diff
@@ -70,7 +70,8 @@
                                TextWrapping="Wrap"
                                Padding="4"
                                Margin="80,1.75,0,0"
-                               Foreground="{StaticResource SearchTextBoxBackground}"
+                               Foreground="{StaticResource autocompletionWindowFont}"
```

Comment:
`autocompletionWindowFont` set to Foreground as color?? The naming seems strange.

Reply:
This was the way I named it in the first PR... happy to rename if needed! What would be a better convention?

Reply:
I would recommend,
1. Use Pascal case 
2. Name it with xxx color, e.g. AutoCompletionWindowTitleColor, etc